README.md

```diff
@@ -136,13 +136,13 @@ let config = [
 
 ## API Reference
 
-### login
+### signin
 
-Start the authorization flow by simply calling `login`. By default, this method uses the values specified in the `Okta.plist` file:
+Start the authorization flow by simply calling `signIn`. By default, this method uses the values specified in the `Okta.plist` file:
 
 ```swift
 OktaAuth
-  .login()
+  .signIn()
```

Comment:
Not sure how I missed this when rewriting the README, but please add `.signIn` to the two code blocks below that immediately jump to `start`:

```diff
OktaAuth
+ .signIn()
  .start(withPListConfig: "CustomPlist", view: self)
  .then { tokens in
    // tokens.accessToken
    // tokens.idToken
    // tokens.refreshToken
  }
  .catch { error in
    // Error
  }
```

Reply:
Done.